cohesity_management_sdk/samples/protection_runs_status/README.md

```diff
@@ -0,0 +1,20 @@
+Samples for Cohesity Management SDK 
```

Comment:
This should be readme about the python file itself, just to keep it simple:
Run parameters
Sample output

Reply:
Done. Readme now looks similar to other samples .